src/reactA11yImgHasAltRule.ts

```diff
@@ -10,17 +10,24 @@ import {
 } from './utils/JsxAttribute';
 import { isJsxSpreadAttribute } from './utils/TypeGuard';
 
-const roleString: string = 'role';
-const altString: string = 'alt';
+const ROLE_STRING: string = 'role';
+const ALT_STRING: string = 'alt';
 
 export function getFailureStringNoAlt(tagName: string): string {
-  return `<${tagName}> elements must have an alt attribute or use role='presentation' for presentational images. \
+  return `<${tagName}> elements must have an non-empty alt attribute or \
+use empty alt attribute and role='presentation' for presentational images. \
 A reference for the presentation role can be found at https://www.w3.org/TR/wai-aria/roles#presentation.`;
 }
 
-export function getFailureStringEmptyAlt(tagName: string): string {
-  return `The value of 'alt' attribute in <${tagName}> tag is undefined or empty. \
-Add more details in 'alt' attribute or use role='presentation' for presentational images. \
+export function getFailureStringEmptyAltAndNotPresentationRole(tagName: string): string {
+  return `The value of alt attribute in <${tagName}> tag is empty and role value is not presentation. \
+Add more details in alt attribute or use empty alt attribute and role='presentation' for presentational images. \
```

Comment:
`Add more details in alt attribute or use empty alt attribute and role='presentation' for presentational images.` better be
`Specify role to equal 'presentation' when 'alt' attribute is empty.`